ilastik/widgets/featureTableWidget.py

```diff
@@ -62,6 +65,44 @@ def __init__(self, text="Sigma"):
         self.setText(text)
 
 
+class _FeatureSelectionColors:
+    """
+    Class that holds some color "constants". Colors change according to
+    used system theme (on windows and MacOS).
+    """
+
+    @staticmethod
```

Comment:
Turn these methods into properties? Unfortunately, `staticmethod(property(meth))` still doesn't work even in Python 3.12, but `classmethod(property(meth))` works just fine.

Also could drop `_color` suffix since it's obvious from the class name.

Reply:
Heya, thanks @emilmelnikov for checking this out! I also wanted to have properties (I wanted something as close to an enum as I could get, I guess, and did have chained classmethod with properties), but it turns out [these are deprecated in py11](https://docs.python.org/3/whatsnew/3.11.html#language-builtins).
Dropping the `_color` makes sense :)

Reply:
Thanks, didn't know this was deprecated!

In this case, it's trivial to roll up a custom descriptor/decorator that does the same thing:

```python
class staticproperty:
    def __init__(self, getter):
        self._getter = getter

    def __get__(self, _instance, _owner=None):
        return self._getter()
```

Probably could be useful in other places too.

Reply:
Hi @emilmelnikov. Thank you again for the suggestion! I guess "trivial" is subjective.

In general I am against introducing syntactic sugar that goes beyond standard Python. I'll keep in in mind for a situation where I feel the benefit is worth introducing this concept that is very likely absent in most other Python code-bases for the sake of removing `()` for function calls. In a way, having an explicit function call here makes it obvious that this value is computed at runtime.